backend/plugins/operation_api/src/modules/tasks/graphql/resolvers/utils.ts

```
@@ -0,0 +1,1132 @@
+  const resolver = resolvers['Task'] || {};
+
+  for (const subResolver of Object.keys(resolver)) {
+    try {
+      additionInfo[subResolver] = await resolver[subResolver](
+        item,
+        {},
+        { models, user },
+        { isSubscription: true },
+      );
+    } catch (unused) {
+      continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)